Xero.NetStandard.OAuth2/Client/Configuration.cs

```diff
@@ -59,7 +59,7 @@ public class Configuration : IReadableConfiguration
                     int ra = response.Headers.TryGetValue("Retry-After", out var ras) && int.TryParse(ras.FirstOrDefault(), out ra)
                         ? ra
                         : 0;
-                    return new ApiException(status, string.Format("Xero API {0} rate limit error calling {1}. Retry-After {2}", limitType, methodName, ra), ra, response.Content, limitType);
+                    return new ApiException(status, string.Format("Xero API {0} rate limit error calling {1}", limitType, methodName), response.Content, ra, limitType);
```

Comment:
This is actually absolutely **critical**.

The current advice provided by Xero regarding capturing retry errors, is to check the existing Exception's message against a regex. If you change the error message, you lose all backwards compatibility.